soarca_fin_python_library/executor.py

```diff
@@ -105,15 +105,15 @@ def _handle_unregister_self_message(self, message: UnregisterSelf):
                 self._wait_for_ack(message.message_id)
                 self.callback(message)
                 return
-            except (Empty, TimeoutError) as e:
+            except (Empty, TimeoutError):
                 self._send_message_as_json(message, topic="soarca")
-            except RuntimeError as e:
```

Comment:
Maybe log the error where you removed the e?

Reply:
Now calls log.debug. In the wait_for_ack method log.info and log.warning are called to provide some additonal context.